README.md

```diff
@@ -1,22 +1,43 @@
-# kubescape.io website
+# kubescape.io Website
 
-This repository contains the source files for the kubescape.io website.
+This repository contains the source files for the [kubescape.io](https://kubescape.io) website. The website is built using [mkdocs](https://www.mkdocs.org/) with the [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) theme.
 
-## Building the site
+## Table of Contents
 
-### Local testing
+- [Getting Started](#getting-started)
+- [Local Testing](#local-testing)
+- [Production](#production)
+- [Contributing](#contributing)
+- [License](#license)
 
-We use [mkdocs](https://www.mkdocs.org/) with the [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) theme. 
+## Getting Started
 
-To build and test locally: 
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
 
-* [install mkdocs-material](https://squidfunk.github.io/mkdocs-material/getting-started/) with `pip` or Docker 
-* run `mkdocs serve`
+### Prerequisites
 
-## Local build
+- Python installed on your local machine
+- pip 
 
-Run `mkdocs build`, and the site will be built in the `site/` directory.
+### Setting Up
 
-### Production
+1. Clone the repository to your local machine.
```

Comment:
this file is cloned to people's local checkout already, they can be reading it there.